b3lb/rest/endpoints/b3lb/endpoints.py

```diff
@@ -191,26 +191,39 @@ async def create(request, endpoint, params, node, secret):
 
     # check if records are enabled
     if secret.is_record_enabled:
-        record_relation = RecordSet()
-        record_relation.secret = secret
-        record_relation.meeting_id = meeting_id
-        record_relation.record_available_url = "https:://{}-{}.{}/{}".format(secret.tenant.slug.lower(), str(secret.sub_id).zfill(3), settings.B3LB_API_BASE_DOMAIN, "b3lb/b/record/available")
-
+        record_ready_url = "https://{}-{}.{}/{}".format(secret.tenant.slug.lower(), str(secret.sub_id).zfill(3), settings.B3LB_API_BASE_DOMAIN, "b3lb/b/record/available")
```

Comment:
Shouldn't the last format parameter ("b3lb/b/record/available") moved into the template string?

Reply:
done